### ISSUE TITLE
Ajv Validation Error Response in JSON

### ISSUE THREAD

**Issue by saasrack** [Romakita/ts-express-decorators#367]
## Informations

Type | Version
---|---
Question | 4.14

****

## Description
I have implemented the `@tsed/ajv` validation package. However, I have hard time to figure out how to response validation errors in JSON format to make it readable to the consuming party. It seems that the `ParseExpressionError` class containing `buildMessage` function is only able to return a pre-set string with some information regarding the validation error.

```javascript
static buildMessage(name: string, expression: string | RegExp | undefined, message?: string) {
    name = name.toLowerCase().replace(/parse|params|filter/gi, "");
    return `Bad request on parameter "request.${name}${expression ? "." + expression : ""}".\n${message}`.trim();
}
```

Is there a way to passthrough the Ajv validation error object or override the response?

Thanks.

**Comment by Romakita** [comment]
Hi @saasrack,

Currently not. But we can work on this, to passthrough the AJV validation error in the response.

I'll work on ASAP.
See you,
Romain

**Comment by Romakita** [comment]
Sorry I've completely forgot to finish this feature. Now is released ;) v4.20.0

**Comment by milewski** [comment]
@Romakita can you show an example on how to use this feature? im also in need of something similar
